alto_tools.py

```diff
@@ -25,7 +25,8 @@ def alto_parse(alto):
     namespace = {'alto-1': 'http://schema.ccs-gmbh.com/ALTO',
                  'alto-2': 'http://www.loc.gov/standards/alto/ns-v2#',
                  'alto-3': 'http://www.loc.gov/standards/alto/ns-v3#',
-                 'alto-4': 'http://www.loc.gov/standards/alto/ns-v4#'}
+                 'alto-4': 'http://www.loc.gov/standards/alto/ns-v4#',
+                 'alto-bnf': 'http://bibnum.bnf.fr/ns/alto_prod'}
```

Comment:
Since the BnF ALTO "dialect" is not an official schema version of ALTO, I would prefer to add a short comment before adding this e.g. 

\#BnF ALTO dialect (unoffical) - for further info see 
\#http://bibnum.bnf.fr/alto_prod/documentation/alto_prod.html